runtime/jnichk/jnicheck.h

```diff
@@ -26,6 +26,7 @@
 #include "jnichk_internal.h"
 
 IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void* reserved);
+J9JavaVM* globalJavaVM;
```

Comment:
This needs an `extern` declaration (in fact, it's probably better to just extern it in the C files that use it to avoid conflict with the actual declaration).

Reply:
added in [47d194e](https://github.com/eclipse-openj9/openj9/commit/47d194ea2e0f95330bdcc673ddb0949d29409dac)